practice_project/main.py

```diff
@@ -33,6 +23,18 @@ def test_is_prime(self):
 		self.assertFalse(is_prime(20))
 		self.assertFalse(is_prime(25))
 		self.assertFalse(is_prime(30))
+
+		self.assertTrue(is_prime(2))
+		self.assertTrue(is_prime(3))
+		self.assertTrue(is_prime(5))
+		self.assertTrue(is_prime(7))
+		self.assertTrue(is_prime(11))
+		self.assertTrue(is_prime(13))
+		self.assertTrue(is_prime(17))
+		self.assertTrue(is_prime(19))
+		self.assertTrue(is_prime(23))
+		self.assertTrue(is_prime(29))
+		self.assertTrue(is_prime(31))
```

Comment:
instead of one big test case, i suggest breaking it down into multiple separate test cases. also look for edge cases (input smaller 0, very large, unexpected format)